src/etc/inc/notices.inc

```diff
@@ -38,15 +38,40 @@ $smtp_authentication_mechanisms = array(
  *	'MD5' => 'MD5'
  *	'CRAM-MD5' => 'CRAM-MD5'
 */
+global $pushover_sounds;
```

Comment:
global declarations must happen only inside functions

Reply:
This is OK in recent PHP versions, it works better to declare a variable global "globally" before use like this.